spec.bs

```diff
@@ -303,6 +303,13 @@ Storage {#storage}
 A user agent holds an <dfn>aggregatable report cache</dfn>, which is a [=list=]
 of [=aggregatable reports=].
 
+Clearing storage {#clearing-storage}
+----------------------------------------
+
+The user agent must expose controls that allow the user to delete data from
```

Comment:
Do we want to mention why the contribution cache data is not cleared? Or should they?

Reply:
That's a good point -- I think we don't usually require this data to be deleted as it's short-lived, but I'm not sure if there's explicit guidance on this. Added a comment along these lines and said the user agent "may" provide controls for them. I'm not sure what's typical here though.